### PR TITLE
Improve CLI configuration documentation. Inspired by #656.

### PR DESCRIPTION
**Pull request by rabbah**
Supersedes #656.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
LGTM

File: docs/README.md [hunk 1]
```diff
@@ -7,11 +7,45 @@ For more details about how OpenWhisk works, see [System overview](./about.md).
...
 
-After OpenWhisk is set up with the CLI, you can begin using it from the command line or through REST APIs.
+- If you cloned the OpenWhisk respository, you will find the CLI in `openwhisk/bin/wsk`.
+
+- Otherwise, download the CLI from an existing deployment.
+You will need to know the base URL for the deployment you want to use and
+install it using [pip](https://pip.pypa.io/).
+
+```
+sudo pip install --upgrade https://{BASE URL}/openwhisk-0.1.0.tar.gz
```
Comment (psuter) [inline docs/README.md]: Using `docker-machine` this failed for me:

    $ sudo pip install --upgrade https://192.168.99.100/openwhisk-0.1.0.tar.gz
    ...
    SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)

...as I would expect. I could bypass with:

    $ sudo pip install --upgrade https://192.168.99.100/openwhisk-0.1.0.tar.gz --trusted-host 192.168.99.100

**Comment by rabbah** [comment]
@psuter added the suggested option in a second commit.

**Comment by psuter** [comment]
Merged as 270e25e0bca86623ca047637414f1f05bd3b8374.

**Pull request closed without merging**